### PR TITLE
Update oidcUtil to allow a callback without userinfo [revisited]

### PR DESCRIPTION
**Pull request by oleksandrpravosudko-okta**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [guidelines](/okta/okta-oidc-js/blob/master/CONTRIBUTING.md#commit)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Adding Tests
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: OKTA-341568

## Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by oleksandrpravosudko-okta** [comment]
intended to replace https://github.com/okta/okta-oidc-js/pull/372

File: packages/oidc-middleware/src/oidcUtil.js [hunk 1]
```diff
@@ -86,13 +86,30 @@ oidcUtil.bootstrapPassportStrategy = context => {
...
-      })
-      : done(null);
+  }, (tokenSet, callbackArg1, callbackArg2) => {
+    let done;
+    let userinfo;
+
+    if (typeof(callbackArg2) !== 'undefined') {
+      done = callbackArg2;
+      userinfo = callbackArg1;
+
```
Comment (denysoblohin-okta) [inline packages/oidc-middleware/src/oidcUtil.js:96]: excess newline

**Review by denysoblohin-okta — APPROVED** [review]
LGTM

**Comment by shuowu** [comment]
@oleksandrpravosudko-okta Can you add the changelog and the version bump in this PR? Thanks

**Review by shuowu — APPROVED** [review]

**Pull request merged**